src/cloneDeep.ts

```diff
@@ -20,9 +28,23 @@ const CLONE_SYMBOLS_FLAG = 4;
  * const deep = cloneDeep(objects)
  * console.log(deep[0] === objects[0])
  * // => false
+ *
+ * // The `skipNativeCheck` flag
+ * const unsupportedNativeObject = { fn: () => 'a' };
+ *
+ * const deep = cloneDeep(unsupportedNativeObject, { skipNativeCheck: true });
+ * console.log(deep === unsupportedNativeObject);
+ * // => false
  */
-function cloneDeep(value) {
-    return baseClone(value, CLONE_DEEP_FLAG | CLONE_SYMBOLS_FLAG);
+function cloneDeep(value, options = {}) {
+    try {
+        if (!options.skipNativeCheck && structuredClone) {
+            return structuredClone(value);
+        }
+        throw new Error('Unsupported structured clone');
```

Comment:
Slightly more accurate, though nothing uses this message so it doesn't really matter.
```suggestion
        throw new Error('structuredClone unsupported or skipped');
```